schema_based_on_diagram.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE patients(
+    id INT PRIMARY KEY, 
+    name VARCHAR(20), 
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+    id INT PRIMARY KEY, 
+    admitted_at TIMESTAMP,
+    status VARCHAR, 
+    patient_id INT,
+    CONSTRAINT fk_patient_id 
+    FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE invoices(
+    id INT PRIMARY KEY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT,
+    CONSTRAINT fk_medical-medical_history_id
```

Comment:
- [ ] I would suggest you use the `underscores` for compound names in order to avoid issues with sql when used dashes, i.e: instead of writing `fk_medical-medical_history_id` you should write `fk_medical_medical_history_id`
- [ ] All "join table" for the many-to-many relastionships is created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)